hack/bundle-automation/generate-bundles.py

```diff
@@ -331,14 +331,14 @@ def injectHelmFlowControl(deployment):
 {{ toYaml . | indent 8 }}
 {{- end }}
 """     
-        if line.strip() == "imagePullSecrets:":
-            lines[i] = """{{- if .Values.global.imagePullSecret }}
```

Comment:
Could you explain the implications of these changes? Is the generation not currently setting things correctly? I was curious so I tried running the `make regenerate` command and it errored out on
```
Updating deployments with antiaffinity, security policies, and tolerations
```
Is that what this addresses?

Reply:
Yes, there were a few changes, but that was the most notable one.
The error you hit is likely bc of line - https://github.com/open-cluster-management/backplane-operator/pull/45/files#diff-de58bfe3d0b76ce51f66a05f7081de09c47abd4d83386a8d7ce2fc0528c21813L385
which is fixed in this PR.

I also updated the helm flow control logic to ensure it matches with the values yaml struct defined here - https://github.com/open-cluster-management/backplane-operator/blob/main/pkg/rendering/renderer.go#L30 so it can regenerate consistently with the same Yaml.

Reply:
We needed to regenerate the charts/crds for this PR to bring in Cluster-managers CRD changes for nodeselector/tolerations.